### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by davidwessman**
- CodeClimate - now calls SimpleCov separately, therefore changing .gitignore
- Recaptcha - now uses different environment variable names, updating
  .env-sample
- No longer using jquery/datepicker
- Devise: uses bypass_sign_in instead

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -375,23 +381,24 @@ GEM
     thread_safe (0.3.5)
     tilt (2.0.5)
     ttfunk (1.4.0)
-    turbolinks (2.5.3)
-      coffee-rails
+    turbolinks (5.0.1)
```
Comment (navro) [inline Gemfile.lock]: Looks like they've added TL5 support to `wice_grid` now 😄
I would still like to test this on `stage` before merging, considering the problems we had on `voting`.

File: app/views/menus/_menu.html.erb [hunk 2]
```diff
@@ -1,28 +1,3 @@
...
-  </li>
-<% else %>
-  <li>
-    <%= link_to(menu, menu_locale_link(menu.link, I18n.locale),
-                'data-no-turbolink': true) %>
-  </li>
-<% end %>
+<li class="<%= menu_class(menu.blank_p) %>">
+  <%= menu_link(menu, I18n.locale) %>
+</li>
```
Comment (navro) [inline app/views/menus/_menu.html.erb]: Much better! 👍

File: Gemfile.lock (hunk 1 above)
Comment (davidwessman) [inline Gemfile.lock]: The new turbolinks totally works better with all our js initiations.
Haven't looked at wice_grid.

Comment (davidwessman) [inline Gemfile.lock]: Live at stage.fsektionen.se btw :)

Comment (navro) [inline Gemfile.lock]: The date picker doesn't work correctly. It needs a full refresh to load... 😒
`wice_grid` works as before.

Comment (navro) [inline Gemfile.lock]: The markdown editor too 😞

Comment (navro) [inline Gemfile.lock]: And the footer script

File: app/assets/javascripts/footer.js [hunk 3]
```diff
@@ -16,4 +16,4 @@ bottom = function () {
     }
 };
 
-$(document).ready(bottom)
+$(document).on('turbolinks:load', bottom);
```
Comment (davidwessman) [inline app/assets/javascripts/footer.js]: @navro Footer fixed

File: app/assets/javascripts/markdown.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        return false;
+      },
+      title: "<%= I18n.t('components.markdown_editor.help', default: 'Markdown Editing Help') %>"
+    };
+    editor = new Markdown.Editor(converter, attr, help);
+    return editor.run();
+  });
+};
+
+$(document).on('turbolinks:load', markdown);
```
Comment (davidwessman) [inline app/assets/javascripts/markdown.js]: @navro Now loads Markdown with own script.

**Comment by davidwessman** [comment]
@navro Should we try it now? :)

**Pull request merged**